src/main/java/uk/ac/ebi/eva/pipeline/configuration/VariantJobsArgs.java

```diff
@@ -53,6 +56,12 @@
 @Component
 public class VariantJobsArgs {
     private static final Logger logger = LoggerFactory.getLogger(VariantJobsArgs.class);
+    private static final String DB_COLLECTIONS_FEATURES_NAME = "db.collections.features.name";
```

Comment:
It's better to reduce the parameter string names across the library and use getter/setters is easier to modify later the different properties if you only have one point that make a search in all the files. Renaming a get/set in java is more practical/easy and clean in general.